src/pages/index.js

```diff
@@ -1,30 +1,91 @@
 import React from "react";
-
 import SEO from "../components/SEO/seo";
-
-import Nav from "../components/nav";
-import Header from "../components/header";
-import Cta from "../components/cta";
-import About from "../components/about";
-import Speakers from "../components/speakers";
-import Sponsors from "../components/sponsors";
-import Gallery from "../components/gallery";
-import Footer from "../components/footer";
+import styled from "styled-components";
+import { logo, sponsorship } from "../components/cloudImages";
 
 function IndexPage() {
   return (
     <>
-      <SEO keywords={["Open Source Community Africa", "OSCA", "Open Source Festival", "OSF"]} title="Home" />
-      <Nav />
-      <Header />
-      <Cta />
-      <About />
-      <Speakers />
-      <Sponsors />
-      <Gallery />
-      <Footer />
+      <SEO
+        keywords={["Open Source Community Africa", "OSCA", "Open Source Festival", "OSF"]}
+        title="OSCAfest 2021 Sponsorship"
```

Comment:
I think the title of this page should be the title of the festival without including "sponsorship" as this will become the actual festival website soon, and we want search engines to do us well. So maybe this should be "Open Source Festival 2021"?